### PR TITLE
Add PostgreSQL help page

### PR DESCRIPTION
**Pull request by TDaglis**

### CODE REVIEW

File: _databases/postgresql.md [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+To connect to your PostgreSQL server from your SourceLair Terminal do the following:
+
+- Type `psql -h postgres -U postgres`
+- Type your password (default `mysecretpassword`)  
+
+If you want to avoid going through the password prompt step every time, there are a few alternatives:
+
+1. Pass the password as an environment variable:  
+`PGPASSWORD=mysecretpassword psql -h postgres -U postgres`
+2. Add a [`.pgpass` file](http://www.postgresql.org/docs/8.3/static/libpq-pgpass.html) in your home directory. This can be achieved with the following command:
```
Comment (akalipetis) [inline _databases/postgresql.md]: Please add the correct version URL - http://www.postgresql.org/docs/9.4/interactive/libpq-pgpass.html

File: _databases/postgresql.md [hunk 2]
```diff
@@ -0,0 +1,37 @@
...
+2. Add a [`.pgpass` file](http://www.postgresql.org/docs/8.3/static/libpq-pgpass.html) in your home directory. This can be achieved with the following command:  
+`echo 'postgres:*:*:postgres:mysecretpassword' >> ~/.pgpass && chmod 0600 ~/.pgpass`  
+After running the following command, `psql -h postgres -U postgres` won't require a password.
+
+After connecting to your server from the Terminal, you can change the default password anytime you want by running:  
+`ALTER USER postgres WITH PASSWORD 'mypass';`
+where `'mypass'` is your desired new password.
+If you are using a `.pgpass` file and change your password, remember to update it in the `.pgpass` file as well.
+
+**Note**: If you connect a new database to an existing project, you will have to reload your Terminal before being able to access it.
```
Comment (akalipetis) [inline _databases/postgresql.md]: Please remove this.

**Comment by akalipetis** [comment]
Apart from minor comments, LGTM :bear:

**Comment by TDaglis** [comment]
Resolved comments

**Comment by akalipetis** [comment]
LGTM :bear:

**Pull request merged**